File: Compilers-2/Mini-Asgn-1_ES18BTECH11028/errorsreporting.py
# Writting a program to show that python shows error line by line

n=1;
if (n > 0) #Here we removed ':' which is a syntax error
    print("Number is postive")
elif n==0  #Here alse we have removed ':' which is also a syntax error
    print("Number is 0")

# But the console will only print the error in line 4
# There will be no error message printed at line 6 as Interpreter
# scans the code one line at a time 

File: Compilers-2/Mini-Asgn-1_ES18BTECH11028/errorsreporting.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int n = 1;
    if n>0  //Here we have forgot to close out expression by brackets()
    {
        cout<<"Number is postive"<<endl;
    }
    else
    {
        cout<<"Number is negative" // Here we didn't end the line with semicolon
    }
    /*As you can see there are errors at line 6,12. When yo run this files the
     console will show you these two errors at a same time*/
    return 0;
}